network/Cargo.toml

```diff
@@ -21,12 +21,16 @@ tokio-util = { version = "0.7.3", features = ["codec"] }
 tonic = { version = "0.7.2", features = ["tls"] }
 tracing = "0.1.36"
 types = { path = "../types" }
+crypto = { path = "../crypto" }
 
 serde = "1.0.144"
 mysten-network = { git = "https://github.com/mystenlabs/mysten-infra.git", rev = "696fa25982acdc87ab64ec6a33cb111edc63a331" }
 workspace-hack = { version = "0.1", path = "../workspace-hack" }
 eyre = "0.6.8"
 
+anemo = { git = "https://github.com/bmwill/anemo.git", rev = "677af751e4c9b5f9822c7d2d5f05aa5f925d871c" }
```

Comment:
Why did you choose to push this part to a separate repo? (I might be missing context here)

Reply:
I've been working on this on the side for a while now, it was easier to have it be somewhere simple for me to work on. I expect it may make sense to move this elsewhere (ie to the mysten org) at some point

Reply:
I do think that it makes sense to be in its own repo, similar to the crypto stuff being separated, though

Reply:
Why not to keep it in `mysten-infra`? No strong opinions, up to you guys.

We'd definitely want to move it to Mysten org to have proper code reviews / CI etc.

Reply:
I've moved the repo to mystenlabs/anemo